histomicstk/deeplab/utils/wsi_dataset_util_large_image.py

```diff
@@ -20,6 +20,22 @@
 ################################ main function #################################
 ################################################################################
 
+def get_image(filename):
+    """
+    Read the inut image and perform channel decomposition if necessary.
+    """
+
+    slide = large_image.open(filename)
+    
+    if slide.frames == 3 and slide.bandCount == 1:
+        slide = large_image.open(filename, style={'bands': [{'framedelta': 0, 'palette': '#f00'},{'framedelta': 1, 'palette': '#0f0'},{'framedelta': 2, 'palette': '#00f'}]})
+    elif slide.frames == 1 and slide.bandCount == 3:
+        pass
+    else:
+        raise Exception('Cannot do channel decomposition with [{}] frame and [{}] band image.'.format(slide.frames, slide.bandCount))

```

Comment:
@sayatmimar doing nit-pick, can you use f-strings rather than format method. Generally, code formatter `Black` package throws errors at `format` as newer Python versions encourage f-strings.